resources/instructions/landingpage.html

```diff
@@ -1,19 +1,29 @@
-<!DOCTYPE html>
```

Comment:
I wouldn't give this page too much scrutiny, as it's going away imminently. I just updated for completeness sake.